plugin-registration.md

```diff
@@ -243,6 +243,20 @@ When you register the back-end navigation you can use [localization strings](loc
 
 To make the sub-menu items visible, you may [set the navigation context](../backend/controllers-ajax#navigation-context) in the back-end controller using the `BackendMenu::setContext` method. This will make the parent menu item active and display the children in the side menu.
 
+Key | Description
+------------- | -------------
+**code** | a string value that acts as an unique identifier for that menu option, required.
+**owner** | a string value that specifies the menu items owner plugin or module in the format "Author.Plugin", optional.
```

Comment:
This also can't be specified in the array configuration AFAIK

Reply:
I see what's confused me, when you output the array `BackendMenu::listSideMenuItems();` it shows `code` but when people write the php code for the sidebar menu, you don't need to write the `'code' => 'example'` so I will remove both of them.

Reply:
I see what's confused me, when you output the array `BackendMenu::listSideMenuItems();` it shows `code` in the array. But when people write the php code for the sidebar menu in the `Pluygins.php` file , you don't need to write the `'owner' => 'example'` so I will remove both of them.

Reply:
It can still be documented here, it just needs to have a disclaimer on where a developer would provide that value.

Reply:
I see, so I'm writing a commit like this:

```
**code** | **this item is auto generated by the cms and should not be added by the developer**, a string value that acts as an unique identifier for that menu option.
**owner** | **this item is auto generated by the cms and should not be added by the developer**, a string value that specifies the menu items owner plugin or module in the format "Author.Plugin".
```